src/restinterface/externalhandler/senderresolver/senderresolver.go

```diff
@@ -30,13 +30,13 @@ import (
 )
 
 const (
-	PROC_TCP      = "/proc/net/tcp"
-	WELLKNOWNPORT = 56001
+	procNetTCP   = "/proc/net/tcp"
+	wellKnowPort = 56001
```

Comment:
This should be revised to `wellKnownPort`.

Reply:
> 
> 
> This should be revised to `wellKnownPort`.

Tell me what you mean? The variable is not used anywhere else in the project

Reply:
I thought you forgot to write `n`. Do you want to use `Know` instead of `Known`?

Reply:
Sorry, I didn't notice! ))) Thank you! )))

Reply:
@t25kim Done! )))